Formula/netdata.rb

```diff
@@ -0,0 +1,62 @@
+class Netdata < Formula
+  desc "Distributed real-time performance and health monitoring"
+  homepage "https://my-netdata.io/"
+  url "https://github.com/firehol/netdata/releases/download/v1.9.0/netdata-1.9.0.tar.bz2"
+  sha256 "542b4799aed1ee03b0a0dbd00fae988a788622c431382e6429362fbbb4f0f017"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "ossp-uuid"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--sysconfdir=#{etc}",
+                          "--localstatedir=#{var}"
+    system "make", "install"
+
+    conf_path = (etc/"netdata")
+    conf_path.mkpath
+    conf_path.install "system/netdata.conf"
+  end
+
+  def post_install
+    inreplace (etc/"netdata/netdata.conf"), /web files owner = .*/, "web files owner = #{ENV["USER"]}"
```

Comment:
Use an `inreplace do |s|` here since you're touching the same file twice, and you'll want to guard this block with a `return if File.read()` so `post_install` doesn't fail if it's run twice.

Reply:
I'm not sure what you mean with:

> you'll want to guard this block with a return if `File.read()` so `post_install` doesn't fail if it's run twice.

Reply:
Something like:

```ruby
def post_install
  config = etc/"netdata/netdata.conf"
  return unless File.exist?(config)
  return if File.read(config).include?("#{ENV["USER"]}")
  inreplace config do |s|
    etc
    etc
  end
end
```

Reply:
config.exist? and config.read should work

Reply:
Updated according to @DomT4's example.